definitions/ext-hardware_security_module/default-dashboard.json

```diff
@@ -0,0 +1,446 @@
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Current CPU",
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "width": 4,
+            "height": 1
+          },
+          
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT max(kentik.snmp.CPU) AS 'CPU Utilization %' WHERE instrumentation.provider = 'kentik'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "WARNING",
+                "value": 90
+              },
+              {
+                "alertSeverity": "CRITICAL",
+                "value": 95
+              }
+            ]
+          }
+        },
+        {
+          "title": "Current Memory",
+          "layout": {
+            "column": 9,
+            "row": 1,
+            "width": 4,
+            "height": 1
+          },
+          
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT max(kentik.snmp.MemoryUtilization) AS 'Memory Utilization %' WHERE instrumentation.provider = 'kentik'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "WARNING",
+                "value": 90
+              },
+              {
+                "alertSeverity": "CRITICAL",
+                "value": 95
+              }
+            ]
+          }
+        },
+        {
+          "title": "CPU Utilization (%)",
+          "layout": {
+            "column": 5,
+            "row": 2,
+            "width": 4,
+            "height": 3
+          },
+          
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT average(kentik.snmp.CPU) as 'Average CPU', min(kentik.snmp.CPU) as 'Min CPU', max(kentik.snmp.CPU) as 'Max CPU' WHERE instrumentation.provider = 'kentik' TIMESERIES 5 MINUTES SLIDE BY MAX "
```

Comment:
@Mesverrum this is probably the only thing blocking this pr.

Reply:
The data is only collected every 5 m by default, so if let it go auto then instead of charts with lines we get dots with gaps.  The behavior of the UI is that when you set the timepicker to a wider range of time then it will automatically expand these intervals to an appropriate timeseries.  You can confirm this by looking at the dashboard for almost ever other network entity definition.  
If you have access to the demotron subaccount you can see the behavior in effect here https://one.newrelic.com/nr1-core/npm-overviews/default/MTYwNjg2MnxFWFR8RklSRVdBTEx8LTMyOTU0ODM4NjY3NTEyMTg5OQ?account=1606862&duration=604800000&state=d4fd0d95-12eb-23c2-c8da-4231675a4ab9

Reply:
TIL.

I think i commented something similar in another PR so just ignore me :D